public/assets/js/scripts.js

```diff
@@ -285,4 +294,131 @@ $(document).ready(function () {
+    let output = [];
+    
+    let popOverIcon = `<a tabindex="0" class="" id="search-ready-awesome-spread" role="button" data-toggle="popover">
+                        <img src="./assets/img/icons/question-mark.svg" alt="payment">
+                      </a>`;
+    let recommendation = '';
+    let recommendationNav = '';
+    let recommendationBody = '';
+    
+    // Recommendations breakdown can be found here - https://docs.google.com/document/d/1gsomOY9qclUz9RzadN3ztJH4Y0ryGT-fukNBFLhJAIU/edit?pli=1#heading=h.yy9lcow7gkem
+    
+    const outrightPersonal = initial >= (avgPropertyCost * 0.5) && balance / periodic < 6 / frequency;
+    const immediatePrivate = initial >= (avgPropertyCost * 0.25) && (avgPropertyCost <= 45000000);
+    const outrightMortgage = initial >= (avgPropertyCost * 0.25);
+    const directSpread = initial >= (avgPropertyCost * 0.2) && balance / periodic > 6 / frequency && balance / periodic <= 24 / frequency;
+    const immediateFederal = initial >= (avgPropertyCost * 0.1) && (avgPropertyCost <= 15000000) && balance / periodic <= 12 / frequency;
+    const spreadFederal = initial >= (avgPropertyCost * 0.01) && (avgPropertyCost <= 15000000) && balance / periodic <= 24 / frequency;
+    const rentToOwn = initial >= (avgPropertyCost * 0.05) && balance / periodic <= 120 / frequency;
+    const spreadPrivate = initial >= (avgPropertyCost * 0.025) && (avgPropertyCost <= 45000000) && balance / periodic <= 24 / frequency;
+    const hybrid = initial >= (avgPropertyCost * 0.01) && balance / periodic <= 24 / frequency;
+    
+
+    if (outrightPersonal || outrightMortgage) {
+      let package = {
+        title: `Outright <img src="./assets/img/icons/question-mark.svg" alt="payment">`,
+        message: 'Start BALLing with an initial deposit of 50%'
+      }
+      output.push(package);
+    } 
+    if (immediatePrivate || immediateFederal || spreadFederal || spreadPrivate) {
+      let package = {
+        title: `Mortgage <img src="./assets/img/icons/question-mark.svg" alt="payment">`,
+        message: 'Start BALLing with an initial deposit of 25%'
+      }
+      output.push(package);
+    } 
+    if (directSpread) {
+      let package = {
+        title: `Spread <img src="./assets/img/icons/question-mark.svg" alt="payment">`,
+        message: 'Start BALLing with an initial deposit of 20%'
+      }
+      output.push(package);
+    } 
+    if (rentToOwn) {
+      let package = {
+        title: `Rent-to-own <img src="./assets/img/icons/question-mark.svg" alt="payment">`,
+        message: 'Start BALLing with an initial deposit of 5%'
+      }
+      output.push(package);
+    }
+    if (hybrid) {
+      let package = {
+        title: `Hybrid <img src="./assets/img/icons/question-mark.svg" alt="payment">`,
+        message: 'Start BALLing with an initial deposit of 1%'
+      }
+      output.push(package);
+    }
+    
+    if ($(window).width() < MOBILE_WIDTH) {
+      for (let i = 0; i < output.length; i++) {    
```

Comment:
I believe the for loop can be placed before the if statement to ensure that we use only 1 `for loop` instead of different ones for mobile and desktop view.